azurelinuxagent/common/event.py

```diff
@@ -342,7 +343,31 @@ def add_metric(self, category, counter, instance, value, log_event=False):
         try:
             self.save_event(json.dumps(data))
         except EventError as e:
-            logger.error("{0}", e)
+            logger.periodic_error(logger.EVERY_FIFTEEN_MINUTES, "[PERIODIC] {0}".format(ustr(e)))
+
+    @staticmethod
+    def _clean_up_message(message):
+        # By the time the message has gotten to this point it is formatted as
+        #
+        #   YYYY/MM/DD HH:mm:ss.fffffff LEVEL <text>.
+        #   YYYY/MM/DD HH:mm:ss.fffffff <text>.
+        #   YYYY/MM/DD HH:mm:ss LEVEL <text>.
+        #   YYYY/MM/DD HH:mm:ss <text>.
+        #
+        # The timestamp and the level are redundant, and should be stripped. The logging library does not schematize
+        # this data, so I am forced to parse the message using a regex.  The format is regular, so the burden is low,
+        # and usability on the telemetry side is high.
+        #
+
+        if not message:
+            return message
+
+        log_format_parser = re.compile(r"^\d+/\d+/\d+\s\d+:\d+:\d+\.?\d*(\s[A-Z]*\s)?\s?(.*)$")
```

Comment:
regex will need to be updated per #1716

Reply:
Done. Updated the tests as well with UTC logs